tendrl/node_agent/node_sync/__init__.py

```diff
@@ -358,6 +359,16 @@ def _run(self):
                         ).save()
                         gevent.sleep(interval)
 
+                # Sync osd journal details
+                Event(
+                    Message(
+                        priority=priority,
+                        publisher=NS.publisher_id,
+                        payload={"message": "node_sync, osd journal details"}
+                    )
+                )
+                osd_sync.sync_osd_journal_details()
```

Comment:
this should ideally go in integrations.ceph and it should have a sync class of its own. node_sync is generic and not related to an sds